.github/actions/create-tf-stand/action.yml

```diff
@@ -21,11 +25,21 @@ runs:
         uses: ./.github/actions/python-requirements
       - name: Deploy stand with terraform
         id: deploy
+        if: inputs.use_real_price == '0'
         shell: bash
         run: |
+          echo "USE PRICE: ${{ inputs.use_real_price }}"
           ssh_key='/tmp/ci-stands'
           echo "${{ inputs.ci_stands_key_hcloud  }}" >> ${ssh_key} && chmod 400 ${ssh_key}
           python3 ./clickfile.py infra deploy --current_branch ${{ github.ref_name}} --head_branch ${{ github.head_ref }} --base_branch ${{ github.base_ref }}
+      - name: Deploy stand with terraform with real price
```

Comment:
why is there no just 1 step deploy and sending --use-real-price as 1 or 0 ? looks like duplication

Reply:
because I added flag to clickfile, but i will change behavior :)